### PR TITLE
feat: add missing sync action for StandalonePrice entity `setPriceTiers`

### PR DESCRIPTION
**Pull request by ARRIOLALEO**
#### Summary
This PR adds support for StandalonePrice `setPriceTiers`,

#### Todo

- Tests
  - [ ] Unit
  - [ ] Integration
  - [ ] Acceptance
- [ ] Documentation
- [ ] `Type` label for the PR

### CODE REVIEW

File: packages/sync-actions/src/prices-actions.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-import {
-  buildBaseAttributesActions,
-} from './utils/common-actions'
-
+import { buildBaseAttributesActions } from './utils/common-actions'
 
 export const baseActionsList = [
   { action: 'changeValue', key: 'value' },
   { action: 'setDiscountedPrice', key: 'discounted' },
+  { action: 'setPriceTiers', key: 'tiers' },
```
Comment (markus-azer) [inline packages/sync-actions/src/prices-actions.js:7]: Thats pretty easy to follow 👍 , and it will do the job, maybe also add todo to add the other two actions `addPriceTier`, `removePriceTier` which will make it more accurate, wdyt?

**Review by markus-azer — APPROVED** [review]
Looks good 👍 , just left one nitpick

**Review by dogayuksel — COMMENTED** [review]
We should add a test case when price tiers on now and then are equal and created sync actions are empty.

**Review by dogayuksel — APPROVED** [review]
